_faq/is-mentor-necessary.md

```diff
@@ -1,6 +1,6 @@
 ---
 question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+I'm interested in that project?"
```

Comment:
Why is this here?

See previous comment https://github.com/coala/projects/pull/678#discussion_r228020574